### PR TITLE
Remove "setCloudControllerManagerConfig" that's not needed in AgentBaker.

### PR DESCRIPTION
**Pull request by yizhang4321**
AgentBaker doesn't use the related cloud controller manager configs.

### CODE REVIEW

File: pkg/agent/datamodel/defaults.go [hunk 1]
```diff
@@ -517,8 +517,6 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 
 		// Configure controller-manager
 		cs.setControllerManagerConfig()
```
Comment (xuto2) [inline pkg/agent/datamodel/defaults.go:519]: this is also not needed

File: pkg/agent/datamodel/defaults.go [hunk 2]
```diff
@@ -517,8 +517,6 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
...
-		// Configure cloud-controller-manager
-		cs.setCloudControllerManagerConfig()
 		// Configure apiserver
 		cs.setAPIServerConfig()
 		// Configure scheduler
```
Comment (xuto2) [inline pkg/agent/datamodel/defaults.go:522]: and this

File: pkg/agent/datamodel/defaults.go (hunk 1 above)
Comment (yizhang4321) [reply pkg/agent/datamodel/defaults.go:519]: It is removed in this PR: https://github.com/Azure/AgentBaker/pull/194

**Comment by xuto2** [comment]
there is conflict

**Comment by yizhang4321** [comment]
Just resolved the conflict.

**Pull request merged**